fearless/Modules/NewWallet/ChainAccountBalanceList/ChainAccountBalanceListViewLayout.swift

```diff
@@ -172,10 +192,62 @@ final class ChainAccountBalanceListViewLayout: UIView {
     private func accountButtonHandler() {
         delegate?.accountButtonDidTap()
     }
+}
 
-    func bind(to viewModel: ChainAccountBalanceListViewModel) {
-        accountNameLabel.text = viewModel.accountName
-        totalBalanceLabel.text = viewModel.balance
-        ethAccountMissingIconImageView.isHidden = !viewModel.ethAccountMissed
+// MARK: - Skeleton
+
+extension ChainAccountBalanceListViewLayout {
+    private func startSkeletonAmimating() {
+        guard skeletonView == nil else {
+            return
+        }
+
+        totalBalanceLabel.alpha = 0
+        setupSkeleton()
+    }
+
+    private func stopSkeletonAnimating() {
+        guard skeletonView != nil else {
+            return
+        }
+
+        skeletonView?.stopSkrulling()
+        skeletonView?.removeFromSuperview()
+        skeletonView = nil
+
+        totalBalanceLabel.alpha = 1
+    }
+
+    private func setupSkeleton() {
+        guard frame.size != .zero else { return }
+        let spaceSize = frame.size
+
+        let skeletonView = Skrull(
+            size: spaceSize,
+            decorations: [],
+            skeletons: createSkeletons(for: spaceSize)
+        )
+        .fillSkeletonStart(R.color.colorSkeletonStart()!)
```

Comment:
Fill start and fill shouldn't be separated from each other. Also it possible that gradient contains more than two colors, so I think it'll be useful to provide array of colors, which can be converted to any gradient

Reply:
This methods implemented in pod SoraUI